examples/Frameworker/Server/Properties/launchSettings.json

```diff
@@ -0,0 +1,12 @@
+﻿{
+  "profiles": {
+    "Server": {
+      "commandName": "Project",
+      "launchBrowser": false,
+      "applicationUrl": "https://localhost:5001",
```

Comment:
Does this (and elsewhere) need to include a non-SSL url as well for completeness?

Reply:
No, the client apps all use TLS.